lib/fastlane/plugin/revenuecat/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -0,0 +1,109 @@
+      def self.run(params)
+        branch = params[:branch]
+        repo_name = params[:repo_name]
+        github_token = params[:github_token]
+        rate_limit_sleep = params[:github_rate_limit]
+        version_number = params[:current_version]
+        files_to_update = params[:files_to_update]
+        files_to_update_without_prerelease_modifiers = params[:files_to_update_without_prerelease_modifiers]
+        changelog_latest_path = params[:changelog_latest_path]
+        changelog_path = params[:changelog_path]
+        editor = params[:editor]
+
+        Helper::RevenuecatHelper.validate_local_config_status_for_bump(branch)
+
+        UI.important("Current version is #{version_number}")
+
+        # Ask for new version number
+        new_version_number = UI.input("New version number: ")
+
+        generated_contents = Helper::RevenuecatHelper.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
+        Helper::RevenuecatHelper.edit_changelog(generated_contents, changelog_latest_path, editor)
+        changelog = File.read(changelog_latest_path)
+
+        Helper::RevenuecatHelper.create_new_release_branch(new_version_number)
+        Helper::RevenuecatHelper.replace_version_number(version_number,
+                                                        new_version_number,
+                                                        files_to_update,
+                                                        files_to_update_without_prerelease_modifiers)
+        Helper::RevenuecatHelper.attach_changelog_to_master(new_version_number, changelog_latest_path, changelog_path)
+        Helper::RevenuecatHelper.commmit_changes_and_push_current_branch("Version bump for #{new_version_number}")
+        Helper::RevenuecatHelper.create_release_pr(new_version_number, changelog)
+      end
+
+      def self.description
+        "Bumps sdk version, asks to update changelog and creates PR with changes."
+      end
+
+      def self.authors
+        ["Toni Rico"]
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :current_version,
+                                       description: "Current version of the sdk",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :changelog_latest_path,
+                                       description: "Path to CHANGELOG.latest.md file",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :changelog_path,
+                                       description: "Path to CHANGELOG.md file",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :files_to_update,
+                                       env_name: "FILES_TO_UPDATE_VERSION",
```

Comment:
Not sure if this is documented anywhere in _fastlane_ (but I should probably add it), but it is best practice for `env_name` to be prefixed with the name of the action so that they don't get overridden or accidentally used by other actions.

I think these naming are probably fine and won't overlap but maybe we could prefix all of these `env_name`s with something like `RC_` (or `RC_INTERNAL`) if we go with a plugin rename?